docu/release_notes/2016-10-28-Technev150.html.md

```diff
@@ -0,0 +1,21 @@
+---
+date: October 28, 2016
+service: 'hybris techne'
+official_version: v1.5.0
+headline: 'Techne v1.5.0 released, New Product Thumbnail component. Revisited Tree pattern.'
+---
+
+<h1>Techne v1.5.0</h1>
+<p>New Product Thumbnail pattern added. Can be used to show a list of products in a grid.</p>
+<p>New Grid Started pages added. Speeds the development of pages used to display products in a grid. It has also our new toolbar for easy filtering and navigation.</p>
```

Comment:
"New Product Grid starter page added to show Product Thumbnails in a Gird"

I think we should just leave it at that. the mention of speed and toolbar may cause confusion and take focus away from the new shiny thing as we already released the toolbar a while ago.